x/bank/spec/02_keepers.md

```diff
@@ -4,9 +4,25 @@ order: 2
 
 # Keepers
 
-The bank module provides three different exported keeper interfaces which can be passed to other modules which need to read or update account balances. Modules should use the least-permissive interface which provides the functionality they require.
+The bank module provides these exported keeper interfaces that can be
+passed to other modules that read or update account balances. Modules
+should use the least-permissive interface that provides the functionality they
+require.
 
-Note that you should always review the `bank` module code to ensure that permissions are limited in the way that you expect.
+Best practices dictate careful review of `bank` module code to ensure that
+permissions are limited in the way that you expect.
+
+## Blacklisting Addresses
+
+The `x/bank` module accepts a map of addresses that are considered blocklisted
+from directly and explicitly receiving funds through means such as `MsgSend` and
+`MsgMultiSend` and direct API calls like `SendCoinsFromModuleToAccount`.
+
+Typically, these addresses are module accounts. If these addresses receive funds
+outside of the expected rules of the state machine, invariants are likely to be
+broken and could result in a halted network.
```

Comment:
Do we have any other example of account being blacklisted, which is not a module account?

Reply:
Nope :)